tests/acceptance/cli_test.py

```diff
@@ -436,6 +436,9 @@ def test_pull_with_ignore_pull_failures(self):
         assert ('repository nonexisting-image not found' in result.stderr or
                 'image library/nonexisting-image:latest not found' in result.stderr)
 
+    def test_pull_with_quiet(self):
+        assert self.dispatch(['pull', '--quiet']).stdout == ''
```

Comment:
I don't think that test is correct - all Compose status messages are written on `stderr`, including progress bars.